packages/dd-trace/test/appsec/rasp/lfi.express.plugin.spec.js

```diff
@@ -82,12 +82,28 @@ describe('RASP - lfi', () => {
         }
       }
 
+      function getAppSync (fn, args, options) {
+        return (req, res) => {
+          try {
+            const result = fn(args)
+            options.onfinish?.(result)
+          } catch (e) {
+            if (e.message === 'DatadogRaspAbortError') {
```

Comment:
I miss a test without this trycatch, to test that we are blocking the request when the exception is handled by express.

Reply:
https://github.com/DataDog/dd-trace-js/pull/4676/files#diff-4947cef62a3c8302aea45bb69bf3612b71b8cc4b3f404dd189d8c32b5dd5a526